src/toolboxes/paintbrush/UI.tsx

```diff
@@ -1,32 +1,16 @@
 import React, { ChangeEvent, ReactElement } from "react";
-import {
-  Tooltip,
-  IconButton,
-  Accordion,
-  AccordionSummary,
-  Typography,
-  AccordionDetails,
-  Grid,
-} from "@material-ui/core";
-
-import {
-  ExpandMore,
-  Brush,
-  RadioButtonUncheckedSharp,
-} from "@material-ui/icons";
-
-import { Tool } from "@/tools";
+import { Popover } from "@material-ui/core";
 
 import { BaseSlider } from "@/components/BaseSlider";
 import { Sliders, SLIDER_CONFIG } from "./configSlider";
 
 import { usePaintbrushStore } from "./Store";
 
 interface Props {
-  expanded: boolean;
-  activeTool: Tool;
-  onChange: (event: ChangeEvent, isExpanded: boolean) => void;
-  activateTool: (tool: Tool) => void;
+  open: boolean;
```

Comment:
Variables that are boolean are conventionally called, e.g. `isOpen`, we should probably do that?